run_docker.py

```diff
@@ -198,6 +198,9 @@ def main(syn, args):
         # expects there to be a folder named /output to write to
         raise Exception("No 'predictions.csv' file written to /output, "
                         "please check inference docker")
+    elif os.stat(os.path.join(output_dir, 'predictions.csv')).st_size == 0:
+        raise Exception("'predictions.csv' file is empty.  Please check inference docker")
```

Comment:
Don't want to raise the exception. We actually want to create a prediction file when one can't be found.  This way we can hide the log files from toil

Reply:
Not sure why empty or missing files isn't handled in the file loader.    The validate_and_score.py code has always checked for a missing file, and is now checking for files that have no rows (empty).      Therefore, it's results.json will give an appropriate reason for the submission being invalid.

mummert@ubntu-v100-x2-todd:~/projects/git/mummert-ibm/objectnet-tools/scoring$ ./validate_and_score.py -f empty -a ~/objectnet/scoring/answers_by_id.json -r
{
  "prediction_file_errors": [
    "Output CSV file contained no valid rows"
  ],
  "prediction_file_status": "INVALID"
}
mummert@ubntu-v100-x2-todd:~/projects/git/mummert-ibm/objectnet-tools/scoring$ ./validate_and_score.py -f does_not_exist -a ~/objectnet/scoring/answers_by_id.json -r
{
  "prediction_file_errors": [
    "Unable to open results file: does_not_exist"
  ],
  "prediction_file_status": "INVALID"
}

Do you want run_docker.py to never generate an exception?